electron/electron/types/whisperTypes.ts

```diff
@@ -110,7 +110,8 @@ export enum whisperLanguages {
   Vietnamese = 'vi',
   Welsh = 'cy',
   Yiddish = 'yi',
-  Yoruba = 'yo'
+  Yoruba = 'yo',
+  unknown = 'unknown'
```

Comment:
This needs to probably be handled better as we are using the whisper list as an input into our localization system